README.md

```diff
@@ -91,7 +91,7 @@ OPTIONS
   --all  see all commands in CLI
 ```
 
-_See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v2.1.4/src/commands/help.ts)_
```

Comment:
This file (README.md) is generated automatically (with `yarn pack`). Hence what you probably want to do is:
- bump `@oclif/plugin-help` to the new version (old version is [here](https://github.com/che-incubator/chectl/blob/master/yarn.lock#L286)): `yarn upgrade @oclif/plugin-help`
- update README.md: `yarn pack`
- update your PR with the new `yarn.lock` and the new `README.md`

Reply:
you mean https://github.com/che-incubator/chectl/blob/1a37091160503e96b380625f5532c72c517f09be/yarn.lock#L286-L288 ?

Reply:
Yes:`yarn.lock` is another file you don't update manually. You need to use the command `yarn upgrade @oclif/plugin-help` instead.

Reply:
ok, I've run the above commands and updated this PR to the resulting changes. Also updated yarn since I was running with an old version.